### PR TITLE
Adding sql quiz questions

### PR DESCRIPTION
**Pull request by KmKalpana**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/agile-quiz.ts [hunk 1]
```diff
@@ -242,6 +242,27 @@ const agileQuiz = [
...
+  {
+    Questions: "FFD Stands for?",
+    Answer: "Feature Driven Development",
+    Distractor1: "Feature Design Development",
+    Distractor2: "Feature Driven Design",
+    Distractor3: "Feature Driven Difference",
+    Explanation:
+      "Feature Driven Development (FDD) involves designing and building features. The difference between FDD and other Agile Development Methodologies is that the features are developed in specific and short phases separately.",
+    Link: "https://www.planview.com/resources/articles/fdd-agile/",
+  },
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:265]: These questions are already in this PR
https://github.com/freeCodeCamp/Developer_Quiz_Site/pull/460

Please remove them here

File: src/data/sql-quiz.ts [hunk 2]
```diff
@@ -598,6 +598,28 @@ const sqlQuiz = [
...
+  {
+    Question:
+      "Which of the following is one of the basic approaches for joining tables?",
+    Answer:"All of the mentioned",
+    Distractor1:"Subqueries",
+    Distractor2:"Union Join",
+    Distractor3:"Natural Join",
+    Explanation:
+        "The SQL subquery is a SELECT query that is embedded in the main SELECT statement. In many cases, a subquery can be used instead of a JOIN.",
+     Link:"https://afteracademy.com/blog/what-is-join-in-dbms-and-what-are-its-types/",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:610]: Please only user freeCodeCamp resources or official documentation here

File: src/data/sql-quiz.ts [hunk 3]
```diff
@@ -598,6 +598,28 @@ const sqlQuiz = [
...
+  {
+    Question:
+      "What type of join is needed when you wish to include rows that do not have matching values?",
+    Answer:"Outer join",
+    Distractor1:"All of the Mentioned",
+    Distractor2:"Equi-join",
+    Distractor3:"Natural Join",
+    Explanation:
+        "OUTER JOIN is the only join which shows the unmatched rows.",
+     Link:"https://mode.com/sql-tutorial/sql-outer-joins/",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:621]: Please only user freeCodeCamp resources or official documentation here

File: src/data/sql-quiz.ts [hunk 4]
```diff
@@ -598,6 +598,28 @@ const sqlQuiz = [
       "Some of the Transaction Control Language commands include COMMIT, ROLLBACK, and SAVE TRANSACTION.",
     Link: "https://www.freecodecamp.org/news/dbms-and-sql-basics/",
   },
+  {
+    Question:
+      "Which of the following is one of the basic approaches for joining tables?",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:603]:
```suggestion
      "In SQL, which of the following is an approach for joining tables?",
```

File: src/data/sql-quiz.ts [hunk 5]
```diff
@@ -598,6 +598,28 @@ const sqlQuiz = [
...
+    Answer:"All of the mentioned",
+    Distractor1:"Subqueries",
+    Distractor2:"Union Join",
+    Distractor3:"Natural Join",
+    Explanation:
+        "The SQL subquery is a SELECT query that is embedded in the main SELECT statement. In many cases, a subquery can be used instead of a JOIN.",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:609]:
```suggestion
        "In SQL, you can use a subquery, union join or natural join to join multiple tables together.",
```

File: src/data/sql-quiz.ts [hunk 6]
```diff
@@ -598,6 +598,28 @@ const sqlQuiz = [
...
+     Link:"https://afteracademy.com/blog/what-is-join-in-dbms-and-what-are-its-types/",
+  },
+  {
+    Question:
+      "What type of join is needed when you wish to include rows that do not have matching values?",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:614]:
```suggestion
      "In SQL, what is the name of the join that will return matched and unmatched values from a table?",
```

File: src/data/sql-quiz.ts [hunk 7]
```diff
@@ -598,6 +598,28 @@ const sqlQuiz = [
...
+    Answer:"Outer join",
+    Distractor1:"All of the Mentioned",
+    Distractor2:"Equi-join",
+    Distractor3:"Natural Join",
+    Explanation:
+        "OUTER JOIN is the only join which shows the unmatched rows.",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:620]:
```suggestion
        "OUTER JOIN is used to return matched and unmatched values from a table.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by KmKalpana** [comment]
modified sql-quiz-questions

File: src/data/agile-quiz.ts (hunk 1 above)
Comment (jdwilkin4) [reply src/data/agile-quiz.ts]: I pushed up a commit to remove these duplicate questions.
But in the future, you want to be extra careful and double check your PR's to make sure you are not adding duplicate content 👍

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**